packages/spcr-settings/settingsSection.tsx

```diff
@@ -109,13 +109,15 @@ class SettingsSection {
     nameId: string,
     description: string,
     defaultValue: string,
+    inputType: string,
```

Comment:
shouldn't it be nullable? `inputType?: string,`

also could you move it 1 line down(below onChange)? To minimize the impact of the breaking changeq.

Reply:
> shouldn't it be nullable? inputType?: string,

I think it shouldn't, it is marked as nullable [here](https://github.com/FlafyDev/spicetify-creator-plugins/pull/6/files#diff-3c6519709b9dd0fbc54d98535cb39c54ddf7338b34d9cc1960715bd6206b9c34R10)

Reply:
I think it should be nullable since I don't want to force whoever uses `addInput` to specify a inputType.

Also since it's now below an optional parameter(onChange) it can't be not nullable